modules/services/event-bridge/variables.tf

```diff
@@ -87,7 +87,8 @@ variable "event_pattern" {
     "Object Restore Initiated",
     "Object Storage Class Changed",
     "Object Tags Added",
-    "Object Tags Deleted"
+    "Object Tags Deleted",
+    "GuardDuty Finding"
```

Comment:
Do we require any docs change for adding this permission by default?

Reply:
that's a good point, @legobrick ?

Reply:
docs will be updated to indicate that we will support guardduty by default